java-maven-sonar-argocd-helm-k8s/spring-boot-app/JenkinsFile

```diff
@@ -21,7 +22,7 @@ pipeline {
     }
     stage('Static Code Analysis') {
       environment {
-        SONAR_URL = "http://34.201.116.83:9000"
+        SONAR_URL = "http://54.91.77.127:9000"
```

Comment:
This should be parameterized. You can refer https://itnext.io/jenkins-tutorial-part-3-parameterized-pipeline-3898643ac6ad#:~:text=Jenkins%20Pipeline%20Parameters%3A&text=All%20of%20them%20should%20be,parameters%20before%20that%20being%20start.